services/classic/management/http.go

```diff
@@ -161,6 +157,10 @@ func (client client) sendRequest(httpClient *http.Client, url, requestType, cont
 				if numberOfRetries == 0 {
 					return nil, azureErr
 				}
+				if response.StatusCode == http.StatusServiceUnavailable {
+					// Wait before retrying the operation
+					time.Sleep(30 * time.Second)
```

Comment:
How about using `client.config.OperationPollInterval` instead of a fixed value?  It defaults to 30 seconds and it user-configurable.

Reply:
Sounds good.  I missed that nuance.  Will add.